nixos/modules/programs/fish.nix

```diff
@@ -29,6 +45,14 @@ in
         type = types.bool;
       };
 
+      translateEnvironment = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          Whether to translate the configured environment to native fish.
```

Comment:
```suggestion
          If enabled, the configured environment will be translated to native fish using <link xlink:href="https://github.com/bouk/babelfish">babelfish</link>. Otherwise, <link xlink:href="https://github.com/oh-my-fish/plugin-foreign-env">foreign-env</link> will be used.
```

Reply:
This is a lot clearer. Thank you!